src/sentry/api/endpoints/organization_discover.py

```diff
@@ -49,6 +61,22 @@ def has_projects_access(self, member, organization, requested_projects):
 
         return set(requested_projects).issubset(set(member_project_list))
 
+
+class OrganizationDiscoverEndpoint(OrganizationMemberEndpoint):
```

Comment:
hmm not sure how i didn't notice this in the previous pr, but i don't think you should be using `OrganizationMemberEndpoint`. you can just use the base `OrganizationEndpoint` and just use `request.user` to filter the projects